### DISCUSSION TITLE
[Feature] Blur face option

### DISCUSSION THREAD

**Discussion by tevadalbnegru** [immich-app/immich#14840] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

It would be interesting if there was an option to blur unwanted faces in photos.
Like, after the faces are detected, just like the "hide person" option, it would be great if there was another option to blur that face.
Honestly, I don’t know what this would entail, so I hope I’m not talking nonsense.
Obviously, it would be much easier than editing each photo in a third-party app to blur them.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bencorrado** [comment]
This would be great for hosting photos of kids groups (like cub scouts) where we have permission to use photos of some people, but not others that we need to blur out.

**Comment by ITSYV** [comment]
Plus one for this.

I'm a photographer and sometimes i shoot sport events. Some parents of kids don't want their kids to be fully visible on those pictures and it's always a hustle to remove them manually. 
It would be great if editors of a shared album could blur them before sending out an album to everyone.

Maybe programming-wise a good idea to blur those faces in the thumbnail versions and keep the original, so it's always possible to revert. This prevents people from downloading a version without the blurs
